src/components/atoms/Loading/index.tsx

```diff
@@ -1,27 +1,62 @@
 "use client";
 
 import { motion } from "framer-motion";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
 import Image from "next/image";
 import React from "react";
-import LogoWhite from "@/images/easytolive/logo/logobranca-fundoprimary.svg";
+import cx from "classnames";
+
+interface ILoadingComponent {
+  fullSize?: boolean;
+  bgColor?: "main" | "secondary" | "none";
+  Icon?: StaticImport;
+}
+
+const LoadingComponent: React.FC<ILoadingComponent> = ({
+  fullSize = true,
+  bgColor = "main",
+  Icon,
+}) => {
+  const backGroundColor = () => {
+    switch (bgColor) {
+      case "main":
+        return "bg-primary-main";
+      case "secondary":
+        return "bg-secondary-main";
+      case "none":
+        return "bg-none";
+      default:
+        return "main";
+    }
+  };
```

Comment:
```suggestion
  const backgroundColor = (color) => {
    switch (color) {
      case "main":
        return "bg-primary-main";
      case "secondary":
        return "bg-secondary-main";
      case "none":
        return "bg-none";
      default:
        return "main";
    }
  };
  
  // call this: 
  // _backgroundColor(bgColor);_
```

This function is so coupling and has another ways to make this a reusable function for all app, using object map for example. But you need written a so good typescript for this. For now, only apply the suggestion and that's good.